app/components/developers/badge_component.rb

```diff
@@ -0,0 +1,14 @@
+module Developers
+  class BadgeComponent < ApplicationComponent
+    private attr_reader :title, :color
+
+    def initialize(title, color:)
+      @title = title
+      @color = color
+    end
+
+    def call
+      tag.span title, class: "inline-flex items-center rounded-md px-2.5 py-0.5 text-sm font-medium bg-#{color}-100 text-#{color}-800"
```

Comment:
Sadly, Tailwind CSS won't be able to pick these up when "parsing" the code [if they are dynamic](https://tailwindcss.com/docs/content-configuration#dynamic-class-names). We need to write out the full class name for each color.

For now, we can do something like this:

```ruby
def call
  tag.span title, class: class_names("inline-flex items-center rounded-md px-2.5 py-0.5 text-sm font-medium", color_classes)
end

def color_classes
  case color.to_sym
  when :blue
    "bg-blue-100 text-blue-100"
  when :green
    "bg-green-100 text-green-100"
  else
    raise "Unknown color: #{color}"
  end
end
```

Reply:
You are right. As I had earlier used direct classes and passed as parameter so required css was already added in my file and did not realize it is not working dynamically.

Will make the necessary changes.